.github/workflows/test.yml

```diff
@@ -328,10 +329,14 @@ jobs:
       run: |
         docker load --input image-pr.tar
     - name: kind setup
+      timeout-minutes: 30
       run: |
         export OVN_IMAGE="ovn-daemonset-f:pr"
         make -C test install-kind
     - name: Run Tests
+      # e2e tests take less than 30 minutes normally, 60 should be more than enough
+      # set 2 1/2 hours for control-plane tests as these might take a while
+      timeout-minutes: ${{ matrix.target == 'control-plane' && 150 || 60 }}
```

Comment:
Does it make sense to limit the kubernetes E2E tests to 60 minutes? During successful tests, these seem to run for 20 to 30 minutes. If 60 mins is too aggressive, it would make sense keeping this at 120 (if it ain't broken, don't fix it?)

Reply:
I think it is fine to set it to 60 for non control plane